_i18n/en.yml

```diff
@@ -1224,6 +1224,9 @@ meta:
   security:
     description: Learn how login.gov keeps personal information private.
     title: Security
+  what-is-login:
+    description: A summary of what login.gov is the service it provides.
```

Comment:
this is just placeholder text, right?

Reply:
Yes, it's placeholder text for `<head><meta name="description">....</meta></head>`.

Reply:
But it still should have been grammatically correct smh